frontend/dockerfile/dockerfile_test.go

```diff
@@ -684,7 +685,54 @@ COPY --from=base2 /foo /f
 	require.NoError(t, err)
 }
 
-func testTarExporter(t *testing.T, sb integration.Sandbox) {
+func testTarExporterBasic(t *testing.T, sb integration.Sandbox) {
+	f := getFrontend(t, sb)
+
+	dockerfile := []byte(integration.UnixOrWindows(
+		`
+FROM scratch
+COPY foo foo
+`,
+		`
+FROM nanoserver
+COPY foo foo
```

Comment:
I wonder why scratch would not work for Windows here?

Reply:
Unfortunately Windows does not have the concept of a "scratch image" as Linux, a valid image must have the system files, and the smallest we have for now is _nanoserver_.